recipe/patch_yaml/arm-variant.yaml

```diff
@@ -0,0 +1,70 @@
+# https://github.com/conda-forge/arm-variant-feedstock/issues/6 arm-variant should be a run
+# dependency not a run constrained. We want to move the dependency from run_constrained to
+# run, but the patcher raises an error if we do that, so we just add the dependency to run.
+if:
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  has_constrains: "arm-variant * sbsa"
+  timestamp_lt: 1748544655000
+then:
+  - add_depends: "arm-variant * sbsa"
+  # - remove_constrains: "arm-variant * sbsa"
+---
+if:
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  has_constrains: "arm-variant * tegra"
+  timestamp_lt: 1748544655000
+then:
+  - add_depends: "arm-variant * tegra"
+  # - remove_constrains: "arm-variant * tegra"
+---
+# https://github.com/conda-forge/cuda-nvcc-impl-feedstock/issues/42 an arm-variant
+# dependency is needed to be added to the outputs of cuda-nvcc-impl-feedstock because
+# run_exports do not work for noarch packages
+if:
+  timestamp_lt: 1748544655000
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  artifact_in:
+    - cuda-crt-dev_linux-aarch64-12.8.93-h579c4fd_2.conda
+    - cuda-crt-tools-12.8.93-h579c4fd_2.conda
+    - cuda-nvcc-dev_linux-aarch64-12.8.93-h4310d6a_2.conda
+    - cuda-nvcc-impl-12.8.93-h614329b_2.conda
+    - cuda-nvcc-tools-12.8.93-h614329b_2.conda
+    - cuda-nvvm-dev_linux-aarch64-12.8.93-h579c4fd_2.conda
+    - cuda-nvvm-impl-12.8.93-h614329b_2.conda
+    - cuda-nvvm-tools-12.8.93-h614329b_2.conda
+then:
+  - add_depends: "arm-variant * sbsa"
+---
+if:
+  timestamp_lt: 1748544655000
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  artifact_in:
+    - cuda-crt-dev_linux-aarch64-12.8.93-h465204a_2.conda
+    - cuda-crt-tools-12.8.93-h465204a_2.conda
+    - cuda-nvcc-dev_linux-aarch64-12.8.93-hef5b290_2.conda
+    - cuda-nvcc-impl-12.8.93-hcab966f_2.conda
+    - cuda-nvcc-tools-12.8.93-h32d051c_2.conda
+    - cuda-nvvm-dev_linux-aarch64-12.8.93-h465204a_2.conda
+    - cuda-nvvm-impl-12.8.93-h32d051c_2.conda
+    - cuda-nvvm-tools-12.8.93-h32d051c_2.conda
+then:
+  - add_depends: "arm-variant * tegra"
+---
+if:
+  timestamp_lt: 1748544655000
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  has_depends: "cuda-version?( *)"
+  not_has_depends: "arm-variant?( *)"
+  not_has_constrains: "arm-variant?( *)"
+then:
+  - add_depends: "arm-variant * sbsa"
```

Comment:
Could we write a glob that combines this with the first condition? That way we can reuse the list in both cases

If not, could we similarly build a list that covers any CUDA Toolkit packages that fall into this case? Hoping this is a relatively small list